modoboa_amavis/lib.py

```diff
@@ -1,5 +1,7 @@
 # coding: utf-8
 
+from __future__ import unicode_literals
```

Comment:
Have you checked there is no side effect with message releasing? (because a socket is used for that)